marklogic-data-hub/src/main/java/com/marklogic/hub/step/impl/MappingStepDefinitionImpl.java

```diff
@@ -40,7 +46,11 @@ public MappingStepDefinitionImpl(String name) {
 
         options.put("sourceDatabase", HubConfig.DEFAULT_STAGING_NAME);
         options.put("targetDatabase", HubConfig.DEFAULT_FINAL_NAME);
-
-        setModulePath("/data-hub/5/builtins/steps/mapping/default/main.sjs");
+        if(versions.isVersionCompatibleWithES()){
```

Comment:
I recommend changing this to do the following:

1. See if BeanUtil has an instance of Versions. Don't throw an error if it doesn't, as there might not be a Spring container in the context that this class is being used (such as a unit test).
1. If it can't get an instance of Versions, use the ES path.
1. Otherwise, use Versions to choose the path. 

I also recommend adding tests on this logic.

Reply:
Fixed in 69488d8bda278aa2ab8b428bab6f587ad18f8b89